pinecone/index_connection_test.go

```diff
@@ -19,60 +19,13 @@ import (
 )
 
 // Integration tests
-func (ts *IntegrationTests) TestNewIndexConnection() {
```

Comment:
These were unit tests which were moved to the bottom of the file and out of the `IntegrationTests` struct.

Reply:
thank yewww